From 53175f76bc060ddcb3390480a5d2f1c3418016d6 Mon Sep 17 00:00:00 2001
Message-ID: <53175f76bc060ddcb3390480a5d2f1c3418016d6.1714488736.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Tue, 30 Apr 2024 16:50:23 +0200
Subject: [PATCH] fix test_mupdf_regressions version check for bug 707727

Bug 707727 is fixed in mupdf's master branch, not in the release branch
and in particular not in 1.24.1. So adjust the check, assuming that the
fix will be backported to 1.24.2
---
 tests/test_mupdf_regressions.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_mupdf_regressions.py b/tests/test_mupdf_regressions.py
index 7d3f3d59..204f5e2c 100644
--- a/tests/test_mupdf_regressions.py
+++ b/tests/test_mupdf_regressions.py
@@ -50,7 +50,7 @@ def test_707727():
     page = doc.reload_page(page)  # required to prevent re-use
     pix1 = page.get_pixmap()
     ok = pix0.samples == pix1.samples
-    if fitz.mupdf_version_tuple >= (1, 24, 1):
+    if fitz.mupdf_version_tuple > (1, 24, 1):
         assert ok
     else:
         assert not ok
-- 
2.45.0.227.g1c88da2b20

